packages/ra-ui-materialui/src/input/AutocompleteInput.js

```diff
@@ -304,12 +314,23 @@ export class AutocompleteInput extends React.Component {
     };
 
     renderSuggestionsContainer = options => {
-        const { containerProps, children } = options;
+        const {
+            containerProps: { className, ...containerProps },
+            children,
+        } = options;
 
         return (
-            <Paper {...containerProps} square>
-                {children}
-            </Paper>
+            <Popper
+                className={className}
+                open
+                anchorEl={this.inputEl}
+                placement="bottom-start"
+                popperOptions={{ positionFixed: true }}
```

Comment:
please use a constant here to avoid [the infamous `{{` effect](https://marmelab.com/blog/2017/02/06/react-is-slow-react-is-fast.html#beware-of-object-literals-in-jsx).

Reply:
Actually, this prop wasn't needed